eras/shelley/impl/cardano-ledger-shelley.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               cardano-ledger-shelley
-version:            1.10.0.0
```

Comment:
Version 1.10.0.0 of shelley is not released, so you don't need to bump the version to 1.11.0.0.0